presto-main/src/main/java/com/facebook/presto/sql/gen/ExpressionCompiler.java

```diff
@@ -93,19 +93,25 @@ public CacheStatsMBean getCursorProcessorCache()
 
     public Supplier<PageProcessor> compilePageProcessor(SqlFunctionProperties sqlFunctionProperties, Optional<RowExpression> filter, List<? extends RowExpression> projections, Optional<String> classNameSuffix)
     {
-        return compilePageProcessor(sqlFunctionProperties, filter, projections, classNameSuffix, OptionalInt.empty());
+        return compilePageProcessor(sqlFunctionProperties, filter, projections, classNameSuffix, false, OptionalInt.empty());
+    }
+
+    public Supplier<PageProcessor> compilePageProcessor(SqlFunctionProperties sqlFunctionProperties, Optional<RowExpression> filter, List<? extends RowExpression> projections, boolean isBlockLargeByteCodeGeneration, Optional<String> classNameSuffix)
```

Comment:
same question: why is there a version that doesn't take the isBlockLargeByteCodeGeneration argument?

Reply:
Done